### DISCUSSION TITLE
Navigate to pages inside TabView

### DISCUSSION THREAD

**Discussion by sylendra7** [xamarin/XamarinCommunityToolkit#1211] [Q&A] [answered]
Is there a way to navigate to another pages inside the tabview ? We already know TabView items accepts only  views, but whenever we navigate to another page we need to show the tab control in the bottom of the page for few screens/pages .

**Comment by WiproSarthak** [comment]
You should use Shell tabs for this scenario
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/tabs
You can show/Hide tabs on any page as per you requirements

**Comment by sylendra7** [comment]
@WiproSarthak  Thing is , my tab is very customizable with fab. So I have used Community toolkit tabview.  If I go for shell I will not get FAB and as well that much customization

**Comment by pictos** [comment, marked as answer]
@sylendra7, right now we can't use Pages with TabView, you can use the [Content Template](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/templates/control-template#substitute-content-into-a-contentpresenter) for this , navigate to other pages, and present the TabView there. 

Since this looks like more a question than a feature request I'll convert this to discussion. If you want a feature request please open a new issue following our template, thanks.

**Comment by overstartup** [comment]
is there any update for this one?
